apps/sms/test/marionette/conversation_test.js

```diff
@@ -84,37 +84,82 @@ marionette('Conversation Panel Tests', function() {
   });
 
   suite('Forward message', function() {
+    var smsBody = 'this is a sms body';
+    var mmsBody = 'this is a mms body';
     setup(function() {
-      messagesApp.launch();
+      var smsThread = ThreadGenerator.generate({
+        body: smsBody
+      });
+
+      var mmsThread = ThreadGenerator.generate({
+        numberOfMessages: 1,
+        messageType: 'mms',
+        participants: ['a@b.c'],
+        attachments: [
+          { type: 'image/png', width: 10,  height: 10 },
+          { type: 'text/plain', content: mmsBody }
+        ]
+      });
 
-      // Set empty messages and contacts store.
-      storage.setMessagesStorage();
+      messagesApp.launch();
+      storage.setMessagesStorage(
+        [smsThread, mmsThread],
+        ThreadGenerator.uniqueMessageId
+      );
+      // empty contact store
       storage.setContactsStorage();
+    });
 
-      messagesApp.Inbox.navigateToComposer();
+    test('Forward a SMS', function() {
+      messagesApp.Inbox.smsConversation.tap();
```

Comment:
__nit:__ could you please use use ```messagesApp.Inbox.findConversation(smsThread.id).tap();``` and ```messagesApp.Inbox.findConversation(mmsThread.id).tap();``` instead? I'm going to get rid of ```firstConversation, smsConversation, mmsConversation``` eventually.